.github/workflows/bump-version.yml

```diff
@@ -0,0 +1,55 @@
+name: Auto Bump Version
+
+on:
+  push:
+    branches:
+      - main  # Trigger on push to main branch
+
+jobs:
+  bump:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+        with:
+            lfs: true
+
+      - name: Set up Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version: '14'
+
+      - run: corepack enable
+
+      - name: Install dependencies
+        run: yarn install --mode skip-build
+        env:
+          # yarn runs in immutable mode "by default" in CI -- turning this off requires an
+          # undocumented env var
+          YARN_ENABLE_IMMUTABLE_INSTALLS: false
+
+      - name: Set up Git
+        run: |
+          git config --global user.name 'github-actions'
+          git config --global user.email 'github-actions@github.com'
+
+      - name: Bump version in root package.json
+        run: yarn version patch
+
+      - name: Bump version in subdirectory package.json
```

Comment:
I think the name could be a little more descriptive, something like: "Bump version of @lichtblick/suite package", instead of general "subdirectory". Just to be a bit more clear for who is reading the pipeline.

Reply:
Changed. I didn't include the name of the repo because I think this info is not necessary